Detectors/Align/include/Align/AliAlgVol.h

```diff
@@ -69,8 +69,6 @@ class AliAlgVol : public TNamed
   //
   const char* GetSymName() const { return GetName(); }
   //
-  Int_t GetVolID() const { return (Int_t)GetUniqueID(); }
-  void SetVolID(Int_t v) { SetUniqueID(v); }
```

Comment:
is this temporal? has the way we reference volumes changes?

Reply:
I've eliminated volD, which was introducing an extra level of referencing. When needed, it can be trivially calculated from the sensorID. 
In some classes I still left them, but they will go also.

Reply:
thanks. makes sense. I agree that there should only be one, obvious way to get things.  If needed we can add a free function that converts them later.